src/Http/Controllers/Corporation/KillmailsController.php

```diff
@@ -24,7 +24,7 @@
 
```

Comment:
# Corporation Killmails
- sorting on victim is throwing an exception
```
	DataTables warning: table id=corporation-killmails - Exception Message:

	SQLSTATE[42S22]: Column not found: 1054 Unknown column 'character_name' in 'order clause' (SQL: select *, `corporation_killmails`.`corporation_id` as `ownerID`, `killmail_victims`.`character_id` as `victimID` from `corporation_killmails` left join `killmail_details` on `corporation_killmails`.`killmail_id` = `killmail_details`.`killmail_id` left join `killmail_victims` on `killmail_victims`.`killmail_id` = `corporation_killmails`.`killmail_id` left join `invTypes` on `killmail_victims`.`ship_type_id` = `invTypes`.`typeID` left join `mapDenormalize` on `killmail_details`.`solar_system_id` = `mapDenormalize`.`itemID` where `corporation_killmails`.`corporation_id` = 98413060 order by `character_name` desc limit 100 offset 0)
```
- sorting on ship type is throwing an exception
```
sorting on ship type is throwing an exception
	DataTables warning: table id=corporation-killmails - Exception Message:

	SQLSTATE[42S22]: Column not found: 1054 Unknown column 'type_name' in 'order clause' (SQL: select *, `corporation_killmails`.`corporation_id` as `ownerID`, `killmail_victims`.`character_id` as `victimID` from `corporation_killmails` left join `killmail_details` on `corporation_killmails`.`killmail_id` = `killmail_details`.`killmail_id` left join `killmail_victims` on `killmail_victims`.`killmail_id` = `corporation_killmails`.`killmail_id` left join `invTypes` on `killmail_victims`.`ship_type_id` = `invTypes`.`typeID` left join `mapDenormalize` on `killmail_details`.`solar_system_id` = `mapDenormalize`.`itemID` where `corporation_killmails`.`corporation_id` = 98413060 order by `type_name` asc limit 100 offset 0)
```

- sorting on location is throwing an exception
```
DataTables warning: table id=corporation-killmails - Exception Message:

	SQLSTATE[42S22]: Column not found: 1054 Unknown column 'item_name' in 'order clause' (SQL: select *, `corporation_killmails`.`corporation_id` as `ownerID`, `killmail_victims`.`character_id` as `victimID` from `corporation_killmails` left join `killmail_details` on `corporation_killmails`.`killmail_id` = `killmail_details`.`killmail_id` left join `killmail_victims` on `killmail_victims`.`killmail_id` = `corporation_killmails`.`killmail_id` left join `invTypes` on `killmail_victims`.`ship_type_id` = `invTypes`.`typeID` left join `mapDenormalize` on `killmail_details`.`solar_system_id` = `mapDenormalize`.`itemID` where `corporation_killmails`.`corporation_id` = 98413060 order by `item_name` asc limit 100 offset 0)
```

- making a research is throwing an exception
```
DataTables warning: table id=corporation-killmails - Exception Message:

	SQLSTATE[42S22]: Column not found: 1054 Unknown column 'corporation_killmails.killmail_time' in 'where clause' (SQL: select count(*) as aggregate from (select '1' as `row_count` from `corporation_killmails` left join `killmail_details` on `corporation_killmails`.`killmail_id` = `killmail_details`.`killmail_id` left join `killmail_victims` on `killmail_victims`.`killmail_id` = `corporation_killmails`.`killmail_id` left join `invTypes` on `killmail_victims`.`ship_type_id` = `invTypes`.`typeID` left join `mapDenormalize` on `killmail_details`.`solar_system_id` = `mapDenormalize`.`itemID` where `corporation_killmails`.`corporation_id` = 98413060 and (LOWER(`corporation_killmails`.`killmail_time`) LIKE %u% or LOWER(`corporation_killmails`.`character_name`) LIKE %u% or LOWER(`corporation_killmails`.`type_name`) LIKE %u% or LOWER(`corporation_killmails`.`item_name`) LIKE %u%)) count_row_table)
```

Reply:
All of these exceptions/findings exists prior to this PR
tested with Version `3.0.8`